tbconnect/tests/test_tasks.py

```diff
@@ -454,3 +459,67 @@ def test_sync_ussd_to_rapidpro(self):
                 },
             },
         )
+
+
+class SendUserDataToCCITests(TestCase):
+    msisdn = "2781234567"
+
+    def test_get_user_profile_data(self):
+        profile = create_user_profile(self.msisdn)
+        response = get_user_profile(self.msisdn)
+
+        self.assertIsNotNone(response)
+        self.assertEqual(response.msisdn, "2781234567")
+        self.assertEqual(profile.province, response.province)
+
+    def test_get_none_existing_user_profile_data(self):
+        response = get_user_profile(self.msisdn)
+
+        self.assertIsNone(response)
+
+    @responses.activate
+    @override_settings(
+        CCI_AUT_URL="https://cci-data-test.com", CCI_AUT_TOKEN="test12345"
+    )
+    def test_send_data_to_cci(self):
+        data = {
+            "msisdn": self.msisdn,
+            "name": "Tom",
+            "language": "Eng",
+            "tb_risk": "High",
+            "responded": "Yes",
+            "tb_tested": "Yes",
+            "tb_test_results": "Yes",
+            "screen_timeStamp": "2023-04-25 13:02:17",
+        }
+
+        responses.add(responses.POST, "https://cci-data-test.com", json=data)
+
+        create_user_profile(self.msisdn)
+        response = send_tbcheck_data_to_cci(data)
+
+        self.assertTrue(response)
```

Comment:
It's truthy but not True. You can leave this check out as we check the string below.

Instead you can test like payload of the request like this: https://github.com/praekeltfoundation/healthcheck/blob/develop/selfswab/tests/test_tasks.py#L104-L124